dist/index.html

```diff
@@ -29,11 +29,10 @@
 
         <!-- Google Analytics: change UA-XXXXX-X to be your site's ID. -->
         <script>
-            (function(b,o,i,l,e,r){b.GoogleAnalyticsObject=l;b[l]||(b[l]=
-            function(){(b[l].q=b[l].q||[]).push(arguments)});b[l].l=+new Date;
-            e=o.createElement(i);r=o.getElementsByTagName(i)[0];
-            e.src='//www.google-analytics.com/analytics.js';
-            r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
+            (function(b,o,i,l){GoogleAnalyticsObject=l;(b[l]=b[l]||
+            function(){(b[l].q=b[l].q||[]).push(arguments)}).l=+new Date;
+            o.getElementsByTagName(i)[0].parentNode.appendChild(o.createElement(i))
+            .src='//www.google-analytics.com/analytics.js'}(window,document,'script','ga'));
```

Comment:
@mathiasbynens Done in https://github.com/h5bp/html5-boilerplate/commit/eee759bfe175e850bbc8e4ad0682ec4fe4bd05d6.